src/index.html

```diff
@@ -12,6 +12,148 @@
   </head>
   <body>
     <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="personal-info">
+        <legend>Personal information</legend>
+        <label for="surname">
+          Surname:
+          <input
+            type="text"
+            autocomplete="disable"
+            name="surname"
+            id="surname">
+        </label>
+        <label for="name">
+          Name:
+          <input
+            type="text"
+            name="name"
+            autocomplete="disable"
+            id="name">
+        </label>
+        <label for="numb">
+          How old are You?
+          <input
+            type="number"
+            value="12"
+            max="100"
+            min="0"
+            name="num"
+            id="numb">
+        </label>
+        <label for="date">
+          Full date of birth:
+          <input
+            type="date"
+            name="date"
+            id="date">
+        </label>
+        <label for="agree">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="agree"
+            id="agree">
+        </label>
+      </fieldset>
+      <fieldset class="registration-info">
+        <legend>Registration</legend>
+        <label for="e-mail">
+          E-mail:
+          <input
+            type="email"
+            id="e-mail"
```

Comment:
```suggestion
            id="email"
```
Щоб не було плутаниці